src/Main.cpp

```diff
@@ -59,6 +59,9 @@ struct Arguments {
 
 			po::notify(vm);
 			return true;
+		} catch (boost::program_options::required_option::error& e) {
+			std::cerr << "A required parameter was not specified!" << "\n"; // make this less vague
+			return false;
```

Comment:
From what little bit of docs gathering I could do this should actually have the name of the argument which is expected? otherwise lgtm

Reply:
Yeah, should of looked into the response required_option errors provide, and see if i could of seen *which* required parameter was not specified, might look into that in the near? future